src/App.jsx

```diff
@@ -14,45 +16,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [sortFiled, setSortFiled] = useState('');
+  const [reverse, setReverse] = useState('');
+
+  function sortAlphabetically() {
+    if (reverse === 'R') {
+      setVisibleGoods([...visibleGoods].sort().reverse());
+    } else {
+      setVisibleGoods([...visibleGoods].sort());
+    }
+
+    setSortFiled('A');
```

Comment:
Consider using constants instead of magic strings
```suggestion
    setSortFiled(ALPHABET);
```